### ISSUE TITLE
Disabled button doesn't pass AA contrast requirement for accessibility

### ISSUE THREAD

**Issue by braven112** [AlaskaAirlines/OrionStatelessComponents__ods-button#21]
# Issue report

Disabled button text doesn't pass AA contrast requirement for accessibility

## Expected Behavior

The button color and text color needs a minimum contrast of 4.5

The button color and text color needs a minimum contrast of 4.5 currently it is at 4.33
## Possible Solution

Darkening the text seems like the best solution in this case since the background of the button is already really light.

## Context

We need to release the routemap and can't have accessibility issues once its in prod.

**Comment by braven112** [comment]
![image](https://user-images.githubusercontent.com/5118118/56615563-2cc19d80-65d0-11e9-958e-a8ac56f38ad4.png)

**Comment by clockeby** [comment]
Several people from the UX/UI team researched this within the larger digital color palette conversation. While creating the specs for various form elements, the specific issue of disabled states was reviewed.

"[1.4.3](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast) Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)

- Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;

- Incidental: **Text or images of text that are part of an inactive user interface component**, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

- Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement."

**Comment by blackfalcon** [comment]
Based on input from @clockeby I am closing this issue.
